### DISCUSSION TITLE
Central curated Database

### DISCUSSION THREAD

**Discussion by kvy1kor** [oss-review-toolkit/ort#6817] [Q&A]
Hello,

We would like to know if there is an already existing central curated database present in opensource, which i can already use and integrate with my ORT setup.

Looking forward to hearing from you.

Thanks in advance

**Comment by sschuberth** [comment]
Among the [supported package curation providers](https://github.com/oss-review-toolkit/ort/tree/main/plugins/package-curation-providers) are [ORT's own "database"](https://github.com/oss-review-toolkit/ort-config/tree/main/curations) (in the form of a Git repository), as well as [ClearlyDefined](https://clearlydefined.io/) that you could both use (also in parallel).

**Comment by kvy1kor** [comment]
Hello @sschuberth, Thanks for the details, but we are more curious to know about ort own database. Are there any plans of having more curation to ORT's own database?

**Reply by sschuberth** [reply]
> Hello @sschuberth, Thanks for the details, but we are more curious to know about ort own database. Are there any plans of having more curation to ORT's own database?

@tsteenbe once had plans to publish a large amount of internal curations to ORT's config repository, but I don't know what happened with that idea.
